File: web/src/resources/R.ts
/*
R.ts
Created by Khrysalis
*/
//! Declares com.lightningkite.template.R
export interface StringInterface {
    already_have_account: string
    anon_account: string
    anon_log_in: string
    app_name: string
    app_subtitle: string
    apple: string
    back: string
    books: string
    check_email: string
    deep_link_intent: string
    deep_link_was_invalid_credentials: string
    deep_link_was_invalid_server: string
    emailAddress: string
    email_sent: string
    enter_email_to_login: string
    generic_error: string
    google: string
    home: string
    i_agree: string
    invalid_email_address: string
    log_in: string
    log_out: string
    my_profile: string
    or: string
    pin: string
    settings: string
    sign_in_apple: string
    sign_in_email: string
    sign_in_github: string
    sign_in_google: string
    sign_up: string
    submit: string
    subscribe: string
    subscription_link: string
    terms_of_service: string
    terms_of_service_body: string
    try_it_out: string
    user_email: string
    welcome_x: string
}
export namespace DefaultStrings {
    export const already_have_account = "I already have an account"
    export const anon_account = "You are not logged in."
    export const anon_log_in = "Show me what it's like"
    export const app_name = "LK Template"
    export const app_subtitle = "Strikingly good software at breakneck speeds."
    export const apple = "Apple"
    export const back = "Back"
    export const books = "Books"
    export const check_email = "A link to log in to the app was sent to your email!"
    export const deep_link_intent = "Deep Link"
    export const deep_link_was_invalid_credentials = "The deep link was invalid. Login credentials did not work."
    export const deep_link_was_invalid_server = "The deep link was invalid. Could not find a server that matched the link."
    export const emailAddress = "Email Address"
    export const email_sent = "Email sent!"
    export const enter_email_to_login = "Enter your email address to log into the app:"
    export const generic_error = "An error occurred"
    export const google = "Google"
    export const home = "Home"
    export const i_agree = "I agree to the terms presented"
    export const invalid_email_address = "Please enter a valid email address."
    export const log_in = "Log In"
    export const log_out = "Log Out"
    export const my_profile = "My Profile"
    export const or = "Or,"
    export const pin = "PIN"
    export const settings = "Settings"
    export const sign_in_apple = "Sign in with Apple"
    export const sign_in_email = "Sign in with Email"
    export const sign_in_github = "Sign in with GitHub"
    export const sign_in_google = "Sign in with Google"
    export const sign_up = "Sign Up"
    export const submit = "Submit"
    export const subscribe = "Subscribe"
    export const subscription_link = "Manage Your Subscription"
    export const terms_of_service = "Terms of Service"
    export const terms_of_service_body = "lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum"
    export const try_it_out = "Try It Out"
    export const user_email = "You are logged in as %1$s"
    export const welcome_x = "\"Welcome, %1$s!\""
}
export const Strings: StringInterface = Object.assign({}, DefaultStrings);
export namespace Colors {
    export const background = "#272727ff"
    export const backgroundAlpha = "#27272780"
    export const barForeground = "#ffffffff"
    export const black = "#000000ff"
    export const colorAccent = "#fcb912ff"
    export const colorAccentDark = "#7e5d0aff"
    export const colorPrimary = "#133e4cff"
    export const colorPrimaryDark = "#0e2f3aff"
    export const disabled = "#888888ff"
    export const foreground = "#ffffffff"
    export const foregroundFade = "#ffffff88"
    export const highlight = "#0000ffff"
    export const ic_launcher_background = "#393939ff"
    export const lightBlue = "#406f9fff"
    export const toggleOff = "#925857ff"
    export const toggleOn = "#58443bff"
    export const transparent = "#00000000"
    export const transparentBack = "#88888888"
}
export namespace Dimen {
    export const corner_radius = "corner_radius"
    export const spacing = "spacing"
    export const spacing2 = "spacing2"
}
import drawable_ic_apple from "./drawables/ic_apple.svg"
import drawable_ic_back from "./drawables/ic_back.svg"
import drawable_ic_back_arrow from "./drawables/ic_back_arrow.svg"
import drawable_ic_close from "./drawables/ic_close.svg"
import drawable_ic_email from "./drawables/ic_email.svg"
import drawable_ic_email_login from "./drawables/ic_email_login.svg"
import drawable_ic_filter from "./drawables/ic_filter.svg"
import drawable_ic_forward from "./drawables/ic_forward.svg"
import drawable_ic_github_mark from "./drawables/ic_github_mark.svg"
import drawable_ic_google from "./drawables/ic_google.svg"
import drawable_ic_home from "./drawables/ic_home.svg"
import drawable_ic_launcher_foreground from "./drawables/ic_launcher_foreground.svg"
import drawable_ic_notification from "./drawables/ic_notification.png"
import drawable_ic_profile from "./drawables/ic_profile.svg"
import drawable_ic_settings from "./drawables/ic_settings.svg"
import drawable_landing_background from "./drawables/landing_background.jpg"
import drawable_logo from "./drawables/logo.svg"
import drawable_logo_title from "./drawables/logo_title.svg"
export namespace Drawables {
    export const button_accent = {name: "button_accent"}
    export const button_background = {name: "button_background"}
    export const button_black = {name: "button_black"}
    export const button_toggle = {name: "button_toggle"}
    export const button_white = {name: "button_white"}
    export const ic_apple = {name: "ic_apple", file: drawable_ic_apple}
    export const ic_back = {name: "ic_back", file: drawable_ic_back}
    export const ic_back_arrow = {name: "ic_back_arrow", file: drawable_ic_back_arrow}
    export const ic_close = {name: "ic_close", file: drawable_ic_close}
    export const ic_email = {name: "ic_email", file: drawable_ic_email}
    export const ic_email_login = {name: "ic_email_login", file: drawable_ic_email_login}
    export const ic_filter = {name: "ic_filter", file: drawable_ic_filter}
    export const ic_forward = {name: "ic_forward", file: drawable_ic_forward}
    export const ic_github_mark = {name: "ic_github_mark", file: drawable_ic_github_mark}
    export const ic_google = {name: "ic_google", file: drawable_ic_google}
    export const ic_home = {name: "ic_home", file: drawable_ic_home}
    export const ic_launcher_foreground = {name: "ic_launcher_foreground", file: drawable_ic_launcher_foreground}
    export const ic_notification = {name: "ic_notification", file: drawable_ic_notification}
    export const ic_profile = {name: "ic_profile", file: drawable_ic_profile}
    export const ic_settings = {name: "ic_settings", file: drawable_ic_settings}
    export const landing_background = {name: "landing_background", file: drawable_landing_background}
    export const launch_background = {name: "launch_background"}
    export const logo = {name: "logo", file: drawable_logo}
    export const logo_title = {name: "logo_title", file: drawable_logo_title}
    export const textedit_background = {name: "textedit_background"}
    export const transparent_button_background = {name: "transparent_button_background"}
}

File: ios/IosProject/resources/R.swift
//
// R.swift
// Created by Khrysalis
//

import Foundation
import UIKit
import XmlToXibRuntime


public enum R {
    public enum drawable {
        public static func ic_apple() -> CALayer { return CAImageLayer(UIImage(named: "ic_apple.png")) }
        public static func ic_back() -> CALayer { return CAImageLayer(UIImage(named: "ic_back.png")) }
        public static func ic_back_arrow() -> CALayer { return CAImageLayer(UIImage(named: "ic_back_arrow.png")) }
        public static func ic_circle_notifications() -> CALayer { return CAImageLayer(UIImage(named: "ic_circle_notifications.png")) }
        public static func ic_close() -> CALayer { return CAImageLayer(UIImage(named: "ic_close.png")) }
        public static func ic_email() -> CALayer { return CAImageLayer(UIImage(named: "ic_email.png")) }
        public static func ic_email_login() -> CALayer { return CAImageLayer(UIImage(named: "ic_email_login.png")) }
        public static func ic_filter() -> CALayer { return CAImageLayer(UIImage(named: "ic_filter.png")) }
        public static func ic_forward() -> CALayer { return CAImageLayer(UIImage(named: "ic_forward.png")) }
        public static func ic_github_mark() -> CALayer { return CAImageLayer(UIImage(named: "ic_github_mark.png")) }
        public static func ic_google() -> CALayer { return CAImageLayer(UIImage(named: "ic_google.png")) }
        public static func ic_home() -> CALayer { return CAImageLayer(UIImage(named: "ic_home.png")) }
        public static func ic_launcher_foreground() -> CALayer { return CAImageLayer(UIImage(named: "ic_launcher_foreground.png")) }
        public static func ic_notification() -> CALayer { return CAImageLayer(UIImage(named: "ic_notification.png")) }
        public static func ic_payment() -> CALayer { return CAImageLayer(UIImage(named: "ic_payment.png")) }
        public static func ic_profile() -> CALayer { return CAImageLayer(UIImage(named: "ic_profile.png")) }
        public static func ic_settings() -> CALayer { return CAImageLayer(UIImage(named: "ic_settings.png")) }
        public static func landing_background() -> CALayer { return CAImageLayer(UIImage(named: "landing_background.jpg")) }
        public static func logo() -> CALayer { return CAImageLayer(UIImage(named: "logo.png")) }
        public static func logo_title() -> CALayer { return CAImageLayer(UIImage(named: "logo_title.png")) }
        public static let allEntries: Dictionary<String, ()->CALayer> = [
            "button_accent": button_accent,
            "button_background": button_background,
            "button_black": button_black,
            "button_toggle": button_toggle,
            "button_white": button_white,
            "ic_apple": ic_apple,
            "ic_back": ic_back,
            "ic_back_arrow": ic_back_arrow,
            "ic_circle_notifications": ic_circle_notifications,
            "ic_close": ic_close,
            "ic_email": ic_email,
            "ic_email_login": ic_email_login,
            "ic_filter": ic_filter,
            "ic_forward": ic_forward,
            "ic_github_mark": ic_github_mark,
            "ic_google": ic_google,
            "ic_home": ic_home,
            "ic_launcher_foreground": ic_launcher_foreground,
            "ic_notification": ic_notification,
            "ic_payment": ic_payment,
            "ic_profile": ic_profile,
            "ic_settings": ic_settings,
            "landing_background": landing_background,
            "launch_background": launch_background,
            "logo": logo,
            "logo_title": logo_title,
            "textedit_background": textedit_background,
            "transparent_button_background": transparent_button_background
        ]
    }
    public enum string {
        public static let already_have_account = NSLocalizedString("I already have an account", comment: "already_have_account")
        public static let anon_account = NSLocalizedString("You are not logged in.", comment: "anon_account")
        public static let anon_log_in = NSLocalizedString("Show me what it's like", comment: "anon_log_in")
        public static let app_name = NSLocalizedString("LK Template", comment: "app_name")
        public static let app_subtitle = NSLocalizedString("Strikingly good software at breakneck speeds.", comment: "app_subtitle")
        public static let apple = NSLocalizedString("Apple", comment: "apple")
        public static let back = NSLocalizedString("Back", comment: "back")
        public static let books = NSLocalizedString("Books", comment: "books")
        public static let check_email = NSLocalizedString("A link to log in to the app was sent to your email!", comment: "check_email")
        public static let deep_link_intent = NSLocalizedString("Deep Link", comment: "deep_link_intent")
        public static let deep_link_was_invalid_credentials = NSLocalizedString("The deep link was invalid. Login credentials did not work.", comment: "deep_link_was_invalid_credentials")
        public static let deep_link_was_invalid_server = NSLocalizedString("The deep link was invalid. Could not find a server that matched the link.", comment: "deep_link_was_invalid_server")
        public static let emailAddress = NSLocalizedString("Email Address", comment: "emailAddress")
        public static let email_sent = NSLocalizedString("Email sent! You can either click the link in your email OR enter the received PIN.", comment: "email_sent")
        public static let enter_email_to_login = NSLocalizedString("Enter your email address to log into the app:", comment: "enter_email_to_login")
        public static let generic_error = NSLocalizedString("An error occurred", comment: "generic_error")
        public static let google = NSLocalizedString("Google", comment: "google")
        public static let home = NSLocalizedString("Home", comment: "home")
        public static let i_agree = NSLocalizedString("I agree to the terms presented", comment: "i_agree")
        public static let invalid_email_address = NSLocalizedString("Please enter a valid email address.", comment: "invalid_email_address")
        public static let log_in = NSLocalizedString("Log In", comment: "log_in")
        public static let log_out = NSLocalizedString("Log Out", comment: "log_out")
        public static let my_profile = NSLocalizedString("My Profile", comment: "my_profile")
        public static let or = NSLocalizedString("Or,", comment: "or")
        public static let pin = NSLocalizedString("PIN", comment: "pin")
        public static let settings = NSLocalizedString("Settings", comment: "settings")
        public static let sign_in_apple = NSLocalizedString("Sign in with Apple", comment: "sign_in_apple")
        public static let sign_in_email = NSLocalizedString("Sign in with Email", comment: "sign_in_email")
        public static let sign_in_github = NSLocalizedString("Sign in with GitHub", comment: "sign_in_github")
        public static let sign_in_google = NSLocalizedString("Sign in with Google", comment: "sign_in_google")
        public static let sign_up = NSLocalizedString("Sign Up", comment: "sign_up")
        public static let submit = NSLocalizedString("Submit", comment: "submit")
        public static let subscribe = NSLocalizedString("Subscribe", comment: "subscribe")
        public static let subscription_link = NSLocalizedString("Manage Your Subscription", comment: "subscription_link")
        public static let terms_of_service = NSLocalizedString("Terms of Service", comment: "terms_of_service")
        public static let terms_of_service_body = NSLocalizedString("lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum", comment: "terms_of_service_body")
        public static let try_it_out = NSLocalizedString("Try It Out", comment: "try_it_out")
        public static let user_email = NSLocalizedString("You are logged in as %1$s", comment: "user_email")
        public static let welcome_x = NSLocalizedString("\"Welcome, %1$s!\"", comment: "welcome_x")
    }
    public enum dimen {
        public static let corner_radius: CGFloat = 8.0
        public static let spacing: CGFloat = 8.0
        public static let spacing2: CGFloat = 16.0
    }
    public enum color {
        public static let background: UIColor = UIColor(named: "color_background")!
        public static let backgroundAlpha: UIColor = UIColor(named: "color_backgroundAlpha")!
        public static let barForeground: UIColor = UIColor(named: "color_barForeground")!
        public static let black: UIColor = UIColor(named: "color_black")!
        public static let colorAccent: UIColor = UIColor(named: "color_colorAccent")!
        public static let colorAccentDark: UIColor = UIColor(named: "color_colorAccentDark")!
        public static let colorPrimary: UIColor = UIColor(named: "color_colorPrimary")!
        public static let colorPrimaryDark: UIColor = UIColor(named: "color_colorPrimaryDark")!
        public static let disabled: UIColor = UIColor(named: "color_disabled")!
        public static let foreground: UIColor = UIColor(named: "color_foreground")!
        public static let foregroundFade: UIColor = UIColor(named: "color_foregroundFade")!
        public static let highlight: UIColor = UIColor(named: "color_highlight")!
        public static let ic_launcher_background: UIColor = UIColor(named: "color_ic_launcher_background")!
        public static let lightBlue: UIColor = UIColor(named: "color_lightBlue")!
        public static let toggleOff: UIColor = UIColor(named: "color_toggleOff")!
        public static let toggleOn: UIColor = UIColor(named: "color_toggleOn")!
        public static let transparent: UIColor = UIColor(named: "color_transparent")!
        public static let transparentBack: UIColor = UIColor(named: "color_transparentBack")!
    }
}
